github_webhook/event_type.py

```diff
@@ -0,0 +1,30 @@
+from enum import Enum
+
+
+class EventType(Enum):
+    """
+    Event enum type
```

Comment:
I think it would be good to link to https://developer.github.com/v3/activity/events/types/ in the docstring so it shows up in the docs, plus anyone keeping this event list up to date in the future knows where to look